src/Model/API/Base/DNSZone/UpdateDNSRecord.php

```diff
@@ -55,6 +57,7 @@ public function getBody(): array
                 new AbstractParameter(name: 'Value', type: Type::STRING_TYPE),
             ]),
             new AbstractParameter(name: 'Comment', type: Type::STRING_TYPE),
+            new AbstractParameter(name: 'Id', type: Type::INT_TYPE, required: true),
```

Comment:
Test this; I remember the `Id` not being necessary for an "update" of the DNS record, but it it's in the OpenAPI specs apparently.